src/Controller/SubscriptionController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Subscription;
+use App\Form\SubscriptionType;
+use App\Repository\SubscriptionRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+#[Route('/subscription')]
+class SubscriptionController extends AbstractController
+{
+    #[Route('/', name: 'app_subscription_index', methods: ['GET'])]
+    public function index(SubscriptionRepository $subscriptionRepository): Response
+    {
+        return $this->render('subscription/index.html.twig', [
+            'subscriptions' => $subscriptionRepository->findAll(),
+        ]);
+    }
+
+    #[Route('/new', name: 'app_subscription_new', methods: ['GET', 'POST'])]
+    public function new(Request $request, EntityManagerInterface $entityManager): Response
+    {
+        $subscription = new Subscription();
+        $form = $this->createForm(SubscriptionType::class, $subscription);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $entityManager->persist($subscription);
+            $entityManager->flush();
+
+            return $this->redirectToRoute('app_subscription_index', [], Response::HTTP_SEE_OTHER);
+        }
+
+        return $this->render('subscription/new.html.twig', [
+            'subscription' => $subscription,
+            'form' => $form,
+        ]);
+    }
+
+    #[Route('/{id}', name: 'app_subscription_show', methods: ['GET'])]
```

Comment:
Does it works ?

Reply:
Yes, I've just add a fix, companies was not visible in the dropdown list